src/server/requirements.ts

```diff
@@ -5,9 +5,10 @@ import * as path from 'path';
 import { ConfigurationTarget, env, ExtensionContext, Uri, window, workspace } from 'vscode';
 import { getJavaagentFlag, getJavaConfiguration, getKey, getXMLConfiguration, IS_WORKSPACE_JDK_ALLOWED, IS_WORKSPACE_JDK_XML_ALLOWED, IS_WORKSPACE_VMARGS_XML_ALLOWED, xmlServerVmargs } from '../settings/settings';
 
-const pathExists = require('path-exists');
-const expandHomeDir = require('expand-home-dir');
-const findJavaHome = require('find-java-home');
+import pathExists = require('path-exists');
+import expandHomeDir = require('expand-home-dir');
+import findJavaHome = require('find-java-home');
```

Comment:
You'll need to remove this when you rebase since we replaced `find-java-home` with `jdk-utils`